scripts/build_test_data

```diff
@@ -15,8 +15,9 @@ invisible(lapply(paths, function(path) {
   }
 }))
 
-out <- system2("./scripts/run_model",
+out <- system2("Rscript",
```

Comment:
This is fine, but `callr::rscript` can make this more pleasant (or even just `callr:r` and drop the docopt wrapper)